uoftscrapers/scrapers/events/__init__.py

```diff
@@ -0,0 +1,126 @@
+from ..scraper import Scraper
+from bs4 import BeautifulSoup, NavigableString
+from datetime import datetime, date
+from collections import OrderedDict
+import urllib.parse as urlparse
+from urllib.parse import urlencode
+import requests
+import pytz
+import json
+import re
+
+class UTEvents:
```

Comment:
We can rename this to `Events` to follow convention (since it is for all 3 campuses, there is no prefix).